.travis.yml

```diff
@@ -1,6 +1,4 @@
 language: scala
-scala:
-- 2.11.6
```

Comment:
scala versions specified in `.travis.yml` create a matrix build. So you would actually see a sub-job for every scala version specified. Which is quite neat, as it would be very easy to see if a build fails with one version and not another. However that introduces a duplication. You have to specify same scala versions in `travis.yml` and in your build definition files.